RUFAS/input_manager.py

```diff
@@ -2086,6 +2101,42 @@ def dump_get_data_logs(self, path: Path) -> None:
         file_path = os.path.join(path, file_name)
         om.dict_to_file_json(self.__get_data_logs_pool, file_path)
 
+    def _validate_metadata(self) -> None:
+        """Checks that top-level metadata has valid and required keys and values."""
+        info_map = {
+            "class": self.__class__.__name__,
+            "function": self._validate_metadata.__name__,
+        }
+        metadata_files = self.__metadata["files"]
+        required_keys = {"path", "type", "properties"}
+        optional_keys = {"title", "description"}
+        valid_keys = required_keys | optional_keys
+        for key, data in metadata_files.items():
+            if missing_keys := (required_keys - data.keys()):
+                om.add_error("Metadata Validation", f"Missing required keys '{missing_keys}' in '{key}'", info_map)
```

Comment:
```suggestion
                om.add_error("Metadata Validation", f"Missing required keys {list(missing_keys)} in '{key}'", info_map)
```
I think this makes the message a little cleaner, and same idea could be applied to the log message below.